documentation/database.md

```diff
@@ -0,0 +1,296 @@
+# Database
+
+> [!WARNING]
+> This documentation is meant to describe the state of the database. The reader should be aware that some undocumented changes may have been done after its last updates, and for this purpose should check the git history of this file.
+>
+> Also contributions are welcome! :heart:
+
+First of all, let's explicit two databases that Grist manages:
+1. The Home Database;
+2. The Document Database (aka the grist document);
+
+The Home database is responsible for things related to the instance, like:
+ - the users and the groups registered on the instance;
+ - the billing;
+ - the organisations (aka sites), the workspaces;
+ - the documents metadata (id, name, workspace under which it is located...);
+ - the rights (ACL) to access to organisations, workspaces and documents (the access to the content of the document is controlled by the document itself);
+
+A Grist Document is a Sqlite database which contains data like:
```

Comment:
I think it's not necessary to specify here that Grist documents are Sqlite database (as you did not specify for Home database), you explain it juste after